index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section class="container">
+      <header>
+        <img src="images/Light.png" alt="status bar" width="100%" />
+        <div class="toolbar">
+          <h1 id="logo">ROZH</h1>
+          <img id="menu-image" src="images/Icon - Menu.png" alt="Menu icon" />
+        </div>
```

Comment:
- [ ] Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. The image you added is unnecessary because this is what the user's phone will display and should not be displayed by the website, so you should remove that.
- [ ] The logo should be wrapped in an `<a>` tag for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. Kindly change that.
- [ ] The hamburger button should be wrapped in a `<nav>` tag for semantics reasons. In the future, you will implement functionality for this button to open a mobile menu but you can leave it as an image by now. I suggest you just wrap the hamburger button around the `<nav>` tag.

Reply:
Thanks so much for the feedback. I implemented all the suggestions.